### PR TITLE
No impact analysis on helpdesk; fixes #6956

### LINKED ISSUE #6956: [9.5] Impact analysis into simplified interface
We see the impact analysis from the simplified interface. It's not wanted, I think

image

### PR DESCRIPTION
**Pull request by AdrienClairembault**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #6956

### CODE REVIEW

**Comment by cedric-anne** [comment]
Maybe you could also add this check on `Impact::displayTabContentForItem()`.

File: inc/ticket.class.php [hunk 1]
```diff
@@ -850,7 +850,9 @@ function defineTabs($options = []) {
       // Enable impact tab if there is a valid linked item
       foreach ($this->getLinkedItems() as $linkedItem) {
          $class = $linkedItem['itemtype'];
-         if (isset($CFG_GLPI['impact_asset_types'][$class])) {
+         if (isset($CFG_GLPI['impact_asset_types'][$class])
+            && Session::getCurrentInterface() !== "helpdesk") {
```
Comment (trasher) [inline inc/ticket.class.php]:
```suggestion
            && Session::getCurrentInterface() === "central") {
```
maybe?

Comment (trasher) [reply inc/ticket.class.php]: To be consistent with "// Impact analysis should not be available outside of central "

Comment (AdrienClairembault) [reply inc/ticket.class.php]: Done :+1:

**Pull request merged**